src/App.jsx

```diff
@@ -6,19 +7,24 @@ import Questions from "./pages/Questions";
 import Quizzes from "./pages/Quizzes";
 import Result from "./pages/Result";
 import Signup from "./pages/Signup";
+import { QuizContext } from "./store/context";
+import { initialState, reducer } from "./store/reducer";
 
 function App() {
+  const [state, dispatch] = useReducer(reducer, initialState);
```

Comment:
try to give a generic name to the state dispatch and reducer function for better readability.

Reply:
Okk